recipes/grpc-google-iam-v1/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "grpc-google-iam-v1" %}
+{% set version = "0.11.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5009e831dcec22f3ff00e89405249d6a838d1449a46ac8224907aa5b0e0b1aec
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --verbose"
+
+requirements:
+  host:
+    - python
```

`host` should almost always have the have the same version requirement on `python` as `run`:
```suggestion
    - python >=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*
```

---

python version constraints appended to host python requirement.